src/TestData.hs

```diff
@@ -1,82 +0,0 @@
-{-# LANGUAGE GeneralizedNewtypeDeriving #-}
```

Comment:
Love the boldness! Pretty sure deleting this will break the test suite today though :)

Reply:
it doesn't break the test-suite, but I realized it will break the server until we fully implement the ReaderT with all the resolvers on board.

Reply:
Fixed